Sources/RichTextKit/Attributes/RichTextAttributeWriter+Alignment.swift

```diff
@@ -28,81 +28,26 @@ public extension RichTextAttributeWriter {
         at range: NSRange
     ) {
         let text = richText.string
-
-        // Text view has selected text
-        if range.length > 0 {
-            return setAlignment(alignment, at: range)
-        }
-
-        // The cursor is at the beginning of the text
-        if range.location == 0 {
-            return setAlignment(alignment, atIndex: 0)
-        }
-
-        // The cursor is immediately after a newline
-        if let char = text.character(at: range.location - 1), char.isNewLineSeparator {
-            return setAlignment(alignment, atIndex: range.location)
-        }
-
-        // The cursor is immediately before a newline
-        if let char = text.character(at: range.location), char.isNewLineSeparator {
-            let location = UInt(range.location)
-            let index = text.findIndexOfCurrentParagraph(from: location)
-            return setAlignment(alignment, atIndex: index)
-        }
-
-        // The cursor is somewhere within a paragraph
-        let location = UInt(range.location)
-        let index = text.findIndexOfCurrentParagraph(from: location)
-        return setAlignment(alignment, atIndex: index)
```

Comment:
This is IMHO needed for proper functionality but it probably needs to tweak a bit with some symbols. I am still wrapping my head around why setting alignment on a new line separated by newLineSeparator changes alignment of previous line as well...